src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -227,6 +227,9 @@ protected void execute() {
             .populateMetadataFields()
             .stewardshipType(StewardshipType.CONTROLLED)
             .accessScope(AccessScope.PRIVATE_ACCESS);
+    if (metadata != null) {
+      metadata.putAll(defaultMetadata(Context.requireWorkspace().getId()));
```

Comment:
Checking my understanding: this will always set the metadata keys `terra-workspace-id` and `terra-cli-server` to appropriate values, including overwriting user-provided values for those keys if they're provided?

I think that's reasonable behavior, but it might be worth documenting or warning users that these are reserved metadata keys (if we don't do that already)